controllers/jobsController.js

```diff
@@ -14,16 +16,17 @@ module.exports = {
   findAllByAssignedTo: function(req, res) {
     
     db.Job
-      .findAll({where:{AssignedToEmail:req.body.email}})
+      .findAll({where:{AssigneeId:req.body.email}})
       .then(dbModel => res.json(dbModel))
       .catch(err => res.status(422).json(err));
   },
 
   getJobs: function(req,res) {
     db.Job
-      .findAll({where:{requestorEmail: req.session.username}})
+      .findAll({include: [{model: db.Child,as: 'assignee'}] ,where:{requestorEmail: req.session.username,status: {[Op.ne]:"Closed"}}})
```

Comment:
can you pull out the object and be like query = {...}; .findAll(query).blah blah blah no one wants to scroll to the right just to read this guy. too many characters in one line also hard to read. declare the object and format it nicely then throw it in the findAll();